pkg/config/linters_config.go

```diff
@@ -35,6 +35,9 @@ type LintersConfig struct {
 	// requiredFields contains configuration for the requiredfields linter.
 	RequiredFields RequiredFieldsConfig `json:"requiredFields"`
 
+	// ssaTags contains configuration for the ssatags linter.
+	SSATags SSATagsConfig `json:"ssaTags"`
```

Comment:
I'm in the middle of a refactor where we are lowercasing these, probably best to lowercase this now to prevent any change later